haystack/components/routers/transformers_text_router.py

```diff
@@ -179,8 +179,7 @@ def from_dict(cls, data: Dict[str, Any]) -> "TransformersTextRouter":
             deserialize_hf_model_kwargs(data["init_parameters"]["huggingface_pipeline_kwargs"])
         return default_from_dict(cls, data)
 
-    @component.output_types(documents=Dict[str, str])
```

Comment:
Removed since like other routers this is defined in the init method since we don't know the names of the outputs ahead of time.

Reply:
This is also the case with zero_shot_text_router [here](https://github.com/deepset-ai/haystack/blob/9dc602ebf3f89e0622e9b5885be545fb5d83f74b/haystack/components/routers/zero_shot_text_router.py#L194). Should that be updated as well?

Reply:
good catch! I'll update that as well